tests/resources/functional/tsql/functions/test_checksum_agg/test_checksum_agg_1.sql

```diff
@@ -0,0 +1,10 @@
+## CHECKSUM_AGG
```

Comment:
we shouldn't have one file per folder, as it'll get very difficult to support with the amount of verifications we want:
- tests/resources/functional/tsql/functions/test_checksum_agg/test_checksum_agg_1.sql
- tests/resources/functional/tsql/functions/test_count/test_count_1.sql
- ...

can we semantically group these files into folders? e.g.
- tests/resources/functional/tsql/functions/checksum_agg_1.sql
- tests/resources/functional/tsql/functions/count_1.sql
- ...

Reply:
I would also prefer not to have so many subdirectories. I merely copied the existing Snowflake layout to keep it the same.